controllers/k8ssandra/schemas.go

```diff
@@ -62,6 +62,14 @@ func (r *K8ssandraClusterReconciler) checkSchemaAgreement(mgmtApi cassandra.Mana
 		return result.Error(err)
 	}
 
+	for uid := range versions {
+		// a key named UNREACHABLE may appear in the results when nodes are unreachable.
+		// We exclude these keys from the check.
+		if uid == "UNREACHABLE" {
```

Comment:
I had to introduce this to allow the operator to make progress when a dc is stopped.

When any node is unreachable, the management api returns a map that looks like below:

```json
{
  "UNREACHABLE": [
    "172.18.0.2"
  ],
  "aa573028-7c70-3b8f-a247-13bbca2011d2": [
    "172.18.0.9",
    "172.18.0.6"
  ]
}
```

If you don't exclude the UNREACHABLE entry, this method will always return an error and the operator will get stuck.

Note: the output of `nodetool describecluster` is similar, so it's not a problem with the management API.